src/components/DocumentationTopic.vue

```diff
@@ -368,6 +361,18 @@ export default {
   @include dynamic-content-container;
 }
 
+.description {
+  &:empty { display: none; }
+
+  &:not(:empty) {
+    margin-bottom: $contenttable-spacing-single-side;
```

Comment:
This could have been put outside the `:not(:empty)`, because we are applying a `display:none` anyway :)